audeer/core/io.py

```diff
@@ -664,7 +709,9 @@ def rmdir(
         NotADirectoryError: if path is not a directory
 
     Examples:
-        >>> mkdir('path1/path2/path3')  # doctest: +SKIP
```

Comment:
After changing to a temporal directory the docstring example failed because the folder `'path1/path2/path3'` did not exist. Took me a while to understand why, but I think `+SKIP` not only skips the output but also the whole expression, i.e. the directory was never created and the example only worked because the folder was created in a previous docstring example.